lib/handler.js

```diff
@@ -76,7 +76,7 @@ internals.handler = function (request, callback) {
         // Check for Error result
 
         if (response.isBoom) {
-            request._log(['handler', 'error'], { msec: timer.elapsed(), error: response.message });
+            request._log(['handler', 'error'], { msec: timer.elapsed(), error: response });
```

Comment:
This would be a breaking change. Instead, just add `data` field with the full response.